py/src/call.py

```diff
@@ -9,6 +9,7 @@
 # used from tests, and the query which asserts that the schema is of expected version.
 EXPECTED_SCHEMA_REVISION = 12
 EXPECTED_CAIRO_VERSION = "0.9.0"
+SUPPORTED_COMMANDS = set(["call", "estimate_fee"])
```

Comment:
```suggestion
SUPPORTED_COMMANDS = {"call", "estimate_fee"}
```

Reply:
Isn't that a dictionary constructor, or is the same curly braces used for both set and dict?

Reply:
No, it's a _set_ constructor. The trick is that if you don't have `key: value` pairs within the curly brackets it's a set, not a dictionary.

Reply:
Changed to frozenset.